run_analysis.py

```diff
@@ -142,8 +275,18 @@ def run_analysis(config, analyses):  # {{{
     if args.generate:
         update_generate(config, args.generate)
 
+    logsDirectory = build_config_full_path(config, 'output',
+                                           'logsSubdirectory')
+    make_directories(logsDirectory)
+
     analyses = build_analysis_list(config)
 
-    run_analysis(config, analyses)
+    parallelTaskCount = config.getWithDefault('execute', 'parallelTaskCount',
+                                              default=1)
+
+    if parallelTaskCount <= 1 or len(analyses) == 1:
+        run_analysis(config, analyses)
+    else:
+        run_parallel_tasks(config, analyses, configFiles, parallelTaskCount)
```

Comment:
If there's only one task, we run that one task as normal.  If there are multiple tasks, we use run_parallel_tasks to launch new calls to `run_analysis.py` but with the `--generate` flag set to run each task on its own in parallel.